iree/base/internal/file_io_win32.cc

```diff
@@ -120,6 +122,8 @@ StatusOr<std::string> GetTempFile(absl::string_view base_name) {
       file_path::JoinPaths(temp_path, base_name) + "XXXXXX";
 
   if (::_mktemp(&template_path[0]) != nullptr) {
+    static std::atomic<int> next_id{0};
```

Comment:
I think this is worth a brief comment why this is necessary. I see the commit description, but would be nice to have it here

Reply:
Added TODO for #3845 to remove it instead.